mmengine/optim/optimizer/optimizer_wrapper.py

```diff
@@ -276,7 +297,12 @@ def param_groups(self) -> List[dict]:
         Returns:
              dict: the ``param_groups`` of :attr:`optimizer`.
         """
-        return self.optimizer.param_groups
+
+        return list(
+            chain(self.optimizer.param_groups,
+                  [self.base_param_settings
+                   ])) if self.base_param_settings is not None else list(
+                       self.optimizer.param_groups)
```

Comment:
Please implement it with multiple lines for the readability

Reply:
no problem, split it by condition